plasmapy/formulary/tests/test_dimensionless.py

```diff
@@ -26,3 +31,21 @@ def test_beta_nan():
     out = beta(T, n, B)
     assert np.isnan(out[1])
     assert out[1].unit == u.dimensionless_unscaled
+
+
+def test_Reynolds_number():
+    r"""Test Reynolds_number in dimensionless.py"""
+    rho = 1490 * u.kg / u.m ** 3
+    v = 0.1 * u.m / u.s
+    L = 0.05 * u.m
+    mu = 10 * u.kg / (u.m * u.s)
+
```

Comment:
While it'd be nice to have a few value tests in here, the doctests already sort of do it, so it's fine.